Gemfile

```diff
@@ -6,7 +6,7 @@ eval(IO.read(File.join(File.dirname(__FILE__), 'Gemfile.local')), binding) if Fi
 
 group :test do
   gem 'rake',                           :require => false unless dependencies.map {|dep| dep.name}.include?('rake')
-  gem 'rspec-puppet',                   :require => false
+  gem 'rspec-puppet', '2.3.2',                   :require => false
```

Comment:
!fix Place an info about this into the code as comment: "TODO: Remove this explicitly pinned version by the time ticket gh-org/repo#XXX is closed".

Also ensure if STRICT_VARIABLES really works.